assets/js/components/Tab/Service/Service.js

```diff
@@ -73,12 +73,14 @@ class ServiceTab extends Component {
             </Col>
           </Row>
         </section>
-        <section>
-          <h2>
-            <FormattedMessage id="app.services.unconnectedHeading" />
-          </h2>
-          <ServiceList services={unusedServices} />
-        </section>
+        {unusedServices.length !== 0 && (
```

Comment:
could use isEmpty instead of array.length !== 0

Reply:
ah, there was another one

Reply:
Weird, I've fixed this and pushed

Reply:
Now it seems to be "fixed".